angular/projects/public-lng/src/app/explore-panel/date-input/date-input.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input, Output, EventEmitter, OnChanges, SimpleChanges } from '@angular/core';
+import { NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
+import { Utils } from 'nrpti-angular-components';
+
+@Component({
+  selector: 'app-date-input',
+  templateUrl: './date-input.component.html',
+  styleUrls: ['./date-input.component.scss']
+})
+export class DateInputComponent implements OnChanges {
```

Comment:
This component seems like a prime candidate for the global library.

Reply:
As mentioned in the stand up yesterday, I was to land this as is.